database/interface.go

```diff
@@ -389,6 +389,21 @@ type Tx interface {
 	// implementations.
 	FetchBlockRegions(regions []BlockRegion) ([][]byte, error)
 
+	// PruneBlocks deletes the block files until it reaches the target size
+	// (specificed in bytes).
+	//
+	// The interface contract guarantees at least the following errors will
+	// be returned (other implementation-specific errors are possible):
+	//   - ErrTxNotWritable if attempted against a read-only transaction
+	//   - ErrTxClosed if the transaction has already been closed
+	//
+	// NOTE: The data returned by this function is only valid during a
```

Comment:
The actual comment on the implementation should be identical to this one IMO.

Reply:
I think it's better to have the comments separate since the interface and the implementation are different. It's also how other comments are done as well in the existing code.